contributing.md

```diff
@@ -0,0 +1,21 @@
+---
+---
+
+# Contributing to How We Have Been Saving The Planet This Week
+
+You can help out by suggesting stories and subjects to [helpdesk@newatoms.com](mailto:helpdesk@newatoms.com). The newsletter and it's site are open source, you can contribute directly on [our GitHub repository](https://github.com/newatoms/hwhbstptw).
+
+## Making a new newsletter on GitHub
+
+1. Make a new branch or fork for this newsletter called `newsletter/` and the issue number, eg: `newsletter/26`.
+1. Make a new newsletter file in the `newsletters` folder, the file should be named `YYYY-MM-DD-` and the issue number and the extension `.md`, eg: `2015-12-12-26.md`.
+2. You can add images to the newsletters folder (with a GIT client or GitHub desktop) to use in your email but please name them connected to the issue, eg: `2015-12-12-26-tesla-roadster.jpg`
+3. When finished you can [open a pull request](https://github.com/newatoms/hwhbstptw/compare?expand=1), this is a centralised place for final checking and where we discuss the issue. On resolving the Pull Request we copy and paste the contents into MailChimp to send it to our subscribers.
```

Comment:
They don't, they can make one but can't pull, they would have to fork I think, but don't completely know how this works at this point with this repo.